mlir/test/e2e/conv2d_host_validation_f16_fwd.toml

```diff
@@ -0,0 +1,178 @@
+directory = "conv2d_host_validation_f16_fwd"
```

Comment:
Comment: it looks like a lot of the tests are the same except for NHWC vs NCHW. Can you factor that out? It might lead to one or two more tests.